PresentScreenings/PresentScreenings.csproj

```diff
@@ -54,17 +54,17 @@
   </ItemGroup>
   <ItemGroup>
     <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\Contents.json" />
-    <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\AppIcon-128.png" />
-    <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\AppIcon-128%402x.png" />
-    <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\AppIcon-16.png" />
-    <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\AppIcon-16%402x.png" />
-    <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\AppIcon-256.png" />
-    <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\AppIcon-256%402x.png" />
-    <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\AppIcon-32.png" />
-    <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\AppIcon-32%402x.png" />
-    <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\AppIcon-512.png" />
-    <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\AppIcon-512%402x.png" />
     <ImageAsset Include="Assets.xcassets\Contents.json" />
+    <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\Mac-16.png" />
+    <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\Mac-16%402x.png" />
+    <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\Mac-32.png" />
+    <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\Mac-32%402x.png" />
+    <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\Mac-128.png" />
+    <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\Mac-128%402x.png" />
+    <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\Mac-256.png" />
+    <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\Mac-256%402x.png" />
+    <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\Mac-512.png" />
+    <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\App Store-512%402x.png" />

```

Comment:
Please consider to take some time to get the new icon actually working ;-)

Reply:
It does seem to work, now the "Mac-*" files are added and committed to the project.